array.go

```diff
@@ -632,3 +632,26 @@ func strToGoIdx(s unistring.String) int {
 	}
 	return int(i)
 }
+
+func (a *arrayObject) MemUsage(ctx *MemUsageContext) (uint64, error) {
+	total := EmptySize
+
+	// inc, err := a.baseObject.MemUsage(ctx)
```

Comment:
maybe? memusage will be handled in a follow up ticket. since this won't be operational until we get that in, I didn't clean up any of this code: https://jira.mongodb.org/browse/REALMC-7452